website/contributing/release-troubleshooting.md

```diff
@@ -42,3 +42,50 @@ git pull <remote> <branch> --tags
 # Make sure you push any local tags you added/removed
 git push <remote> <branch> --follow-tags
 ```
+
+### The Tag Has Been Pushed but NPM Failed to Publish React Native
+
+The release process takes various steps, but they can be summarised with these two:
+
+1. Create a version tag on the GitHub repository
+2. Publish all the artifacts on Maven and NPM
+
+Step 2 takes some time because it has to build everything we need to publish and it can happen that some builds fail or there are other issues.
```

Comment:
```suggestion
Step 2 takes most of the CI time because it has to build all the artifacts and prefabs needed - so, it might happen that CI will fail during this part, hence not reaching the publish to npm step.
```